### ISSUE TITLE
multipolygon invisible

### ISSUE THREAD

**Issue by Mahabarata** [openstreetmap/openstreetmap-website#2100]
Hi,

I have 2 pbs, one with ID and the other one with OSM. I open 2 issues, with the same text because my "story" mixes both.
I wanted to add a relation to a polygon (in ID) : the relation is a very big one (scrub) and I'm in the middle, very far from the outer and from any inner. Visible here :
https://www.openstreetmap.org/query?lat=-27.70305&lon=-66.00780

1) when I select the polygon and go in its relations' part, the scrub is not listed : when zooming out (losing the possibility to edit the polygon), I found the outer line, found the number of the relation. But when I come back to the polygon (zoom in) and put this number in the relation, nothing happens. How to add a relation if the wanted relation is not listed and you can not add its number ?
2) Before to zoom, I opened a new tab (right click on the title "openstreetmap") to be out of ID but in the area of the polygon (see link above). I use the last button on the right (request on the objets) on the scrub. Result : the request don't see the scrub. The request sees the administrative limits and nothing else. But on the map we see only (or almost) thing : the scrub, lol. Even if I found how to have the number of the scrub (see point 1), I think it should be better that the request find it directly.

Best regards
Mahab

**Comment by tomhughes** [comment]
We show whatever overpass reports, so if you have an issue then you need to take it up there.

**Comment by Mahabarata** [comment]
Ok, I didn't know that was an overpass's pb.
Thanks for your help.
Best regards

**Comment by mmd-osm** [comment]
> it sounds here like your question is really "how to" rather than any sort of bug report

Right, it was a how to question. Behaviour is by design, no bug to fix.

**Comment by Mahabarata** [comment]
Euh, no, not a question in fact : for me, the query should give the scrub
If you read my first message, I found how to have the number of the scrub before to open this issue so it is not my question.
Best regards
